src/Service/Security/PackageScanner/SensioLabsPackageScanner.php

```diff
@@ -69,18 +77,17 @@ private function saveResult(Package $package, string $status, array $result): vo
     {
         $date = new \DateTimeImmutable();
         $package->setScanResult($status, $date, $result);
+        /**
+         * @var string
+         */
+        $packageName = $package->name();
         $this->results->add(new ScanResult(Uuid::uuid4(), $package, $date, $status, $result));
-    }
-
-    /**
-     * @param array<string,array<string,string>> $error
-     */
-    private function saveError(Package $package, array $error): void
-    {
-        $status = ScanResult::STATUS_ERROR;
-        $date = new \DateTimeImmutable();
-        $package->setScanResult(ScanResult::STATUS_ERROR, $date, $error);
-        $this->results->add(new ScanResult(Uuid::uuid4(), $package, $date, $status, $error));
+        $this->messageBus->dispatch(new SendScanResult(
```

Comment:
I don't think that an email should be sent every time. It would be better to do it only for errors - in that case when security vulnerability occurs.